themes/bootstrap3/js/facets.js

```diff
@@ -1,87 +1,143 @@
-          ? facet.count.toString().replace(/\B(?=(\d{3})+\b)/g, separator) : null,
-        excludeUrl: allowExclude && !facet.isApplied ? excludeUrl : '',
-        excludeTitle: excludeTitle
+    // Create facet text element
+    const orFacet = facet.operator === 'OR';
+    let textEl;
+    if (orFacet) {
+      const iconLabel = el("span", "facet-value icon-link__label");
+      iconLabel.innerText = facet.displayText;
+
+      textEl = el("span", "text");
+      textEl.innerHTML = facet.isApplied
+        ? VuFind.icon("facet-checked", { title: VuFind.translate("Selected"), class: "icon-link__icon" })
+        : VuFind.icon("facet-unchecked", "icon-link__icon");
+      textEl.append(iconLabel);
+    } else {
+      textEl = el("span", "text");
+      textEl.append(facet.displayText);
+    }
+
+    // Create link element
+    const linkEl = el("a", (orFacet ? " icon-link" : ""));
+    linkEl.setAttribute("href", currentPath + facet.href);
+    linkEl.setAttribute("title", facet.displayText);
+    linkEl.append(textEl);
+
+    // Create facet element
+    const classes = "facet js-facet-item"
+      + (facet.isApplied ? " active" : "")
+      + (orFacet ? " facetOR" : " facetAND");
+    let facetEl;
+    if (excludeEl) {
+      linkEl.className += " text";
+      facetEl = el("div", classes);
+      facetEl.append(linkEl);
+      if (badgeEl) {
+        facetEl.append(badgeEl);
       }
-    });
+      facetEl.append(excludeEl);
+    } else {
+      if (badgeEl) {
+        linkEl.append(badgeEl);
+      }
+      linkEl.className = classes + " " + linkEl.className;
+      facetEl = linkEl;
+    }
+
+    // Create toggle button
+    const toggleButton = el("button", "facet-tree__toggle-open");
+    toggleButton.setAttribute('aria-expanded', facet.hasAppliedChildren ? 'true' : 'false');
+    toggleButton.setAttribute('data-toggle-aria-expanded', '');
+    toggleButton.setAttribute('aria-label', facet.displayText);
+
+    let itemContainerEl = el("span", "facet-tree__item-container" + (allowExclude ? " facet-tree__item-container--exclude" : ""));
+    itemContainerEl.append(facetEl);
+
+    // Create an li node with or without children
+    const liEl = el("li");
+    if (hasChildren) {
+      liEl.className = "facet-tree__parent";
+      const childUlId = 'facet_' + facetName + '_' + (++counter.count);
+
+      toggleButton.setAttribute('aria-controls', childUlId);
+      toggleButton.innerHTML = VuFind.icon("facet-opened", "facet-tree__opened") + VuFind.icon("facet-closed", "facet-tree__closed");
+
+      const childrenEl = buildFacetNodes(facetName, facet.children, currentPath, allowExclude, excludeTitle, showCounts, counter, locale);
+      childrenEl.id = childUlId;
+
+      liEl.append(toggleButton, itemContainerEl, childrenEl);
+    } else {
+      toggleButton.innerHTML = VuFind.icon("facet-noncollapsible", "facet-tree__noncollapsible");
+      toggleButton.setAttribute('disabled', '');
```

Comment:
Ah, yes. I changed most to single ones since they were prevalent in the file, but we could discuss this more generally. I find single quotes easier to read, and because I write a lot of PHP code as well, they come naturally. But either way is fine for me. Anyway, I wouldn't do any more than

Reply:
That last sentence seems to just trail off... ;-)

Reply:
Oops. :) ... I wouldn't do any more than absolutely required here to get things working, so we want to standardize on single or double quotes, I'd do that in another task. And preferably with a test that any future changes follow the choice.